### ISSUE TITLE
Don't migrate the tickets multiple times

### ISSUE THREAD

**Issue by Zlopez** [gridhead/protop2g#34]
When running the protop2g multiple times it creates the tickets every time (for example https://gitlab.com/testgroup519/protop2g_test/-/issues/?sort=created_date&state=opened&search=11393&first_page_size=20). It should just sync the existing ticket instead of creating a new one.

**Comment by gridhead** [comment]
@Zlopez, thanks for reporting the issue.

While this can be added as a feature to this project, I would recommend using one of the [many options](https://github.com/gridhead/protop2g#migrate-issue-tickets) available to filter out and specifically choose the tickets that should be migrated from the source namespace to the destination namespace. Pull requests are welcome for additional options.

This can also be complemented with a helper script that probes into the existing set of issue tickets in the destination repository and provides a complement set (i.e. identities of the tickets that are not available on the destination namespace) to the program and then, exclusively those tickets would be moved.

**Comment by jwflory** [comment]
I agree that this would be useful. Remembering where an in-progress migration left off would make the tool easier to use.
